devops/scripts/env_to_yaml_config.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+set -o nounset
+# set -o xtrace
+#
+# Usage:
+#    load_env.sh <.env file>
+#
+
+cp config.sample.yaml config.yaml
+# Loop over the relevant lines in the file specified in $1 (passed in after the loop)
+# The loop source filters the lines in the source file to those that should be treated
+# as variable definitions
+for f in "devops/auth.env" "devops/.env" "templates/core/.env"
+do
+  while read -r line
+  do
+    # split the line into name/value
+    name=$(echo "$line" | cut -d= -f1| tr '[:upper:]' '[:lower:]')
```

Comment:
how do you skip/ignore commented lines?

Reply:
at line 30 we source the file without the comments